internal_ws/ykcompile/src/lib.rs

```diff
@@ -633,13 +633,6 @@ impl TraceCompiler {
         self.restore_regs(&*CALLER_SAVED_REGS);
     }
 
-    /// Emit a NOP operation.
```

Comment:
This a debugging aid. It can be useful to insert `nop`s as waypoints when starting at traces.

Reply:
While this is true, I just tend to put the `nop`s directly into the other dynasm calls. So I think we can remove this.